src/components/DpTootip/DpTooltip.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <component
+    :is="nodeType"
+    class="inline-block">
+    <slot />
```

Comment:
The `VPopover` component uses the default slot for the tooltip trigger, too, but it also exposes another slot named "popover" which receives the tooltip content.

Reply:
true. But we don't use it. thats why I thought we can keek it simple. I also did not enable passing all the fancy options. In my opinion we can enhance it, when we need it.

Reply:
> But we don't use it.

I found several occurrences of the slot with a search for `<template v-slot:popover>`.

Reply:
its within here 94d01bcb47d5a69010ef6b0ff0fe85eefa86e94e
I could't test it by now because I have a problem with dplan and don't know how to use slots in storybook propperly

Reply:
> I could't test it by now because I have a problem with dplan and don't know how to use slots in storybook propperly

Neither can i. Let's check this when demosplan can be accessed as usual, again!